lib/phases/cold-launch.js

```diff
@@ -86,27 +87,29 @@ ColdLaunch.prototype.capture = function() {
   this.dispatcher.on(Phase.PERFORMANCEENTRY, function(entry) {
     phase.debugEventEntry(Phase.PERFORMANCEENTRY, entry);
 
+    var perfEntry = entry;
+
     var ignore = phase.fullyLoaded &&
-        entry.entryType === 'mark' &&
```

Comment:
It was calling it out on what is now line 129: originally was entry.entryPoint = entryPoint;

Note to self: For future lint fixes, attach the original lint output to the bug to make it easier for the reviewer to see corresponding fixes! #mybad